Document/0x06g-Testing-Network-Communication.md

```diff
@@ -209,6 +209,8 @@ Certificate pinning is the process of associating the mobile app with a particul
 The certificate can be pinned during development, or at the time the app first connects to the backend.
 In that case, the certificate associated or 'pinned' to the host at when it seen for the first time. This second variant is slightly less secure, as an attacker intercepting the initial connection could inject their own certificate.
 
+Certificate pinning is a good security practice and should be used for all applications that handle sensitive information. [EFF's Observatory](https://www.eff.org/pl/observatory) lists the root and intermediate CAs that major operating systems automatically trust. Please refer to the [map of the roughly 650 organizations that are Certificate Authorities Mozilla or Microsoft trust (directly or indirectly)](https://www.eff.org/files/colour_map_of_CAs.pdf "Map of the 650-odd organizations that function as Certificate Authorities trusted (directly or indirectly) by Mozilla or Microsoft"). Use certificate pinning if you don't trust at least one of these CAs.
```

Comment:
The first 2 sentences do not seem to make sense together. soo why is it a good security practice? What are you trying to say with this paragraph?

Reply:
I removed the whole paragraph as it's pointing to very old information. I added the SSLmate link in the section before to point out breaches in CAs which should encourage to use SSL Pinning to reduced the attack surface

Reply:
I added another paragraph at the beginning. Let me know what you think.